src/base/Notifier.sol

```diff
@@ -9,6 +9,7 @@ import {PositionInfo} from "../libraries/PositionInfoLibrary.sol";
 
 /// @notice Notifier is used to opt in to sending updates to external contracts about position modifications or transfers
 abstract contract Notifier is INotifier {
+    using CustomRevert for address;
     using CustomRevert for bytes4;
```

Comment:
I'd personally just do 
```
    using CustomRevert for *;
```

instead of 2 different lines

Reply:
or only keep `using CustomRevert for bytes4` and call `bubbleUpAndRevertWith` as a normal function